packages/components/kbd/src/Kbd.tsx

```diff
@@ -0,0 +1,9 @@
+import { ComponentPropsWithoutRef, forwardRef, PropsWithChildren } from 'react'
+
+import { kbdStyles } from './Kbd.styles'
```

Comment:
Imho it's unnecessary to split styles from markup for simpler components like this one. This is one of the benefit of using Tailwind.

Reply:
I prefer being very strict with the DS rules and decoupling the view from the logic. WDYT?

Reply:
done!